framework/decode/vulkan_stats_consumer.h

```diff
@@ -57,6 +60,7 @@ class VulkanStatsConsumer : public gfxrecon::decode::VulkanConsumer, public gfxr
     uint64_t                        GetMaxAllocationSize() const { return max_allocation_size_; }
     uint64_t                        GetAnnotationCount() const { return annotation_count_; }
     const std::vector<std::string>& GetOperationAnnotationDatas() const { return operation_annotation_datas_; }
+    const auto&                     GetResolutions() const { return resolutions_; }
```

Comment:
I'm not a fan of `auto` here where it forces the future contributors to scroll down the file to know what the function they might want to call actually returns (`std::unordered_set<vk::Extent2D>`).